lib/ExtUtils/MakeMaker.pm

```diff
@@ -609,7 +628,7 @@ END
         elsif (
             $cmr
                 ? !$cmr->accepts_module($prereq, $pr_version)
-                : $required_version > $pr_version
+                : $required_version lt $pr_version
```

Comment:
This reverses the conditional, breaking it.  The switch from numeric to stringy comparison is also unexplained in the commit message or anywhere else.  And I'm pretty sure we want to stick with numeric comparisons here.